maro/simulator/scenarios/oncall_routing/business_engine.py

```diff
@@ -250,11 +257,23 @@ def step(self, tick: int) -> None:
 
         # Interrupt and throw decision event
         if (tick + 1) % self._config["interrupt_cycle"] == 0 and len(self._oncall_order_buffer) > 0:
+            route_meta_info_dict = {}
+            for route in self._routes:
+                t = self._current_dest_arrival_tick[route.name]
```

Comment:
Estimated one should be used here? (not the actual one)

Maybe it's better to use the actual coordinate to calculate the live estimated duration to the next stop